docs/_scripts/notebook_hooks.py

```diff
@@ -127,6 +127,30 @@
     "how-tos/human_in_the_loop/breakpoints.md": "how-tos/human_in_the_loop/add-human-in-the-loop.md",
     "cloud/how-tos/human_in_the_loop_breakpoint.md": "cloud/how-tos/add-human-in-the-loop.md",
     "how-tos/human_in_the_loop/edit-graph-state.ipynb": "how-tos/human_in_the_loop/time-travel.md",
+    # LGP migration-related redirects - once LG is also migrated, we can add a redirect for the whole site
+    "concepts/langgraph_platform.md": "https://docs.langchain.com/langgraph-platform/langgraph-platform",
+    "concepts/langgraph_components.md": "https://docs.langchain.com/langgraph-platform/langgraph-components",
+    "concepts/langgraph_server.md": "https://docs.langchain.com/langgraph-platform/langgraph-server",
+    "concepts/langgraph_studio.md": "https://docs.langchain.com/langgraph-platform/langgraph-studio",
+    "cloud/how-tos/invoke_studio.md": "https://docs.langchain.com/langgraph-platform/invoke-studio",
+    "concepts/langgraph_cli.md": "https://docs.langchain.com/langgraph-platform/cli",
+    "cloud/how-tos/studio/quick_start.md": "https://docs.langchain.com/langgraph-platform/quick-start-studio",
+    "concepts/sdk.md": "https://docs.langchain.com/langgraph-platform/sdk",
+    "concepts/auth.md": "https://docs.langchain.com/langgraph-platform/auth",
+    "concepts/assistants.md": "https://docs.langchain.com/langgraph-platform/assistants",
+    "concepts/deployment_options.md": "https://docs.langchain.com/langgraph-platform/deployment-options",
+    "cloud/quick_start.md": "https://docs.langchain.com/langgraph-platform/deployment-quickstart",
+    "cloud/deployment/setup.md": "https://docs.langchain.com/langgraph-platform/setup-app-requirements-txt",
+    "concepts/langgraph_self_hosted_data_plane.md": "https://docs.langchain.com/langgraph-platform/langgraph-self-hosted-data-plane",
```

Comment:
For these deployment pages, should we remove the extra "langgraph-" before the deployment type?